resources/authgear/templates/en/translation.json

```diff
@@ -966,6 +966,7 @@
   "v2-error-server": "Server encountered an internal error.",
   "v2-error-screen-title": "Unexpected Issues",
   "v2-error.error-password-policy-violated": "Invalid password format.",
+  "v2-error-phone-number-format": "ncorrect phone number format for the selected country.",
```

Comment:
typo

Reply:
Fixed in 07d9746dcca35529d34a99ae42726ac3ab8acb21